src/ophys_etl/workflows/db/schemas.py

```diff
@@ -203,6 +203,40 @@ def _generate_binary_mask(
             ] = True
         return mask.tolist()
 
+    def is_valid(self, include_decrosstalk: bool = False) -> bool:
+        """Return True if no processing flags (motion_border, decrosstalk)
+        flags are set.
+
+        Parameters
+        ----------
+        include_decrosstalk : bool Optional
+            Check decrosstalk flags for is_valid. Decrosstalk is required
+            for mesoscope experiments imaging multiple experiments at once.
+            Other experiments using different equipment do not require
+            decrosstalk.
+
+        Returns
+        -------
+        is_valid : bool
+            Composite flag that returns True if no processing flags are set.
+            False if any flag is set.
+        """
+        if self.is_in_motion_border:
+            return False
+        if include_decrosstalk:
+            if self.is_decrosstalk_ghost:
+                return False
+            elif self.is_decrosstalk_invalid_raw_active:
+                return False
+            elif self.is_decrosstalk_invalid_raw:
+                return False
+            elif self.is_decrosstalk_invalid_unmixed:
+                return False
+            elif self.is_decrosstalk_invalid_unmixed_active:
```

Comment:
if `include_decrosstalk` and none of the flags are set, then this will incorrectly return None. Please add a test for this.

Probably cleaner to do something like

```
flag_set = [self._is_in_motion_border]
if include_decrosstalk:
    flag_set += [self._is_decrosstalk_ghost, ...]
return sum(flag_set) > 0
```